docs/landing-zone-v2/scripts/README.md

```diff
@@ -0,0 +1,149 @@
+# Deployment of KRM Landing Zone v2
+
+## Table of Contents
+<!-- vscode-markdown-toc -->
+- [KRM Landing Zone v2](#krm-landing-zone-v2)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Implementation](#implementation)
+
+## <a name='Introduction'></a>Introduction
+
+This solution for those using GitHub as their Git repository, deploys the Landing Zone specified in https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/tree/main/docs/landing-zone-v2#option-1---org-level-folder
+
+The script provided creates:
+- Initial Organization configuration.
+- A GCP folder to host the landing zone hierarchy.
+- A config controller project.
+- A VPC, a subnet and a Cloud NAT.
+- Private Service Connect to access googleapis.com and gcr.io
+- VPC firewall rules.
+- A config controller cluster.
+- IAM permission for the "Yakima" service account.
+- A Private GitHub Repository
+- Create your Landing zone (Gatekeeper policies & Core-landing-zone OR Experimentation)
+
+## <a name='Implementation'></a>Implementation
+
+1. Clone Repo and get into the Bootstrap directory
+
+Once this Repository has been cloned to your local repository, run the following command.
+
+```shell
+git clone https://github.com/yw-liftandshift/KCC-Landing-Zone.git
```

Comment:
Why is clone being used here instead of using `kpt pkg get` to acquire the versioned packages? Looks like this would just be pulling from `main`. If it's just for the bootstrap script we should probably start packaging the script as part of a release or an independent release so we can tag it instead of grabbing it from `main` directly.